SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,73 @@
+header:
+  expiration-date: '2025-07-04T00:00:00.000Z'
+  project-url: 'https://github.com/karmada-io/karmada'
+  schema-version: '1.0.0'
+  last-updated: '2024-07-04'
+  last-reviewed: '2024-07-04'
+  changelog: 'https://github.com/karmada-io/karmada/tree/master/docs/CHANGELOG'
+  license: 'https://github.com/karmada-io/karmada/blob/master/LICENSE'
+project-lifecycle:
+  status: 'active'
+  bug-fixes-only: False
+  core-maintainers:
+    - 'https://github.com/karmada-io/karmada/blob/master/MAINTAINERS.md'
+  roadmap: 'https://github.com/karmada-io/karmada/blob/master/ROADMAP.md'
+  release-process: 'https://karmada.io/docs/releases'
+contribution-policy:
+  accepts-pull-requests: True
+  accepts-automated-pull-requests: True
+  automated-tools-list:
+    - automated-tool: 'dependabot'
+      action: 'allowed'
+      path:
+        - /
+        - /cluster/images/
+  contributing-policy: 'https://github.com/karmada-io/community/blob/main/CONTRIBUTING.md'
+  code-of-conduct: 'https://github.com/karmada-io/community/blob/main/CODE_OF_CONDUCT.md'
+documentation:
+  - 'https://karmada.io/'
+distribution-points:
+  - 'https://hub.docker.com/u/karmada'
+  - 'https://github.com/karmada-io/karmada/releases'
+security-testing:
+  - tool-type: 'sca'
+    tool-name: 'Dependabot'
+    tool-version: '2'
+    integration:
+      ad-hoc: False
+      ci: False
+      before-release: False
+    tool-rulesets:
+      - 'https://github.com/karmada-io/karmada/blob/master/.github/dependabot.yml'
+    tool-url: 'https://github.com/dependabot'
+  - tool-type: 'sca'
+    tool-name: 'trivy'
+    tool-version: 'v0.52.2'
+    integration:
+      ad-hoc: False
+      ci: True
+      before-release: False
+    tool-rulesets:
+      - 'https://github.com/karmada-io/karmada/blob/master/.github/workflows/ci-image-scanning.yaml'
+    tool-url: 'https://github.com/aquasecurity/trivy-action'
+    comment: 'trivy-action is the github action for trivy. The current version of trivy-acton is v0.23.0 and the version of trivy used is v0.52.2.'
```

Comment:
Do we need to update this file often? When dependabot automatically updates trivy-action.

Reply:
> Do we need to update this file often? 

This file is meant to be updated and reviewed manually as needed, at least once a year. 

> When dependabot automatically updates trivy-action.

I've seen some implementations that modify `SECURITY-INSIGHTS.yml` by automatically submiting pr, and given that trivy-action isn't updated very often, I think we can manually update `SECURITY-INSIGHTS.yml` after trivy-action is updated.

Reply:
Good question. I'm concerned about this, manually maintaining this might always outdated.

Reply:
I can modify the comment to reduce the need for manual updates